pandas/core/arrays/sparse.py

```diff
@@ -2207,6 +2219,7 @@ def to_coo(self):
         import_optional_dependency("scipy")
         from scipy.sparse import coo_matrix
 
+        self._parent.sparse._validate_sparse()
```

Comment:
Is there a reason that here `self._parent` is needed while above you use `self._validate_sparse()`

Reply:
Indeed, it is not necessary